### ISSUE TITLE
NVDA (with Firefox) announces the incorrect number of menuitemradio elements in a list when using display:inline-block in CSS

### ISSUE THREAD

**Issue by graemecoleman** [nvaccess/nvda#8151]
I've run into what I think might be a potential bug whereby NVDA (in Firefox) announces the incorrect number of elements provided with a role of menuitembutton - specifically when the CSS display property is set to inline-block. I have attached a mocked-up example to demonstrate the issue.

[menubuttonradio_example.zip](https://github.com/nvaccess/nvda/files/1889544/menubuttonradio_example.zip)

### Steps to reproduce:
1. Open the above file
2. Using the Tab key, navigate to the first button on the page, immediately below the "Class applied to a container span element" heading and provided with an accessible name of "First".
3. Again using the Tab / Shift + Tab keys, navigate through each of the buttons. 

### Expected behavior
NVDA should announce each button, followed by the position of the button in the list and the total list size (e.g. "First, radio menu item, (un)checked, 1 of 4", "Second, radio menu item, (un)checked, 2 of 4" etc.).

### Actual behavior
NVDA announces each button correctly, but does not announce the position of the button in the list or the number of buttons in the list correctly; each button is announced as "First, radio menu item, (un)checked, 1 of 1" "Second, radio menu item, (un)checked, 1 of 1" and so on.

### System configuration
NVDA 2018.1.1 (installed), Mozilla Firefox ESR 52.7.3 (64-bit), Mozilla Firefox Developer Edition 60.0b10 (64-bit).

### Windows version
Windows 10 Home 10.0.16299 Build 16299 x64-based PC (not in a VM)

### Other information
The issue appears to occur when the buttons are wrapped inside span elements. If the related CSS class is applied directly to the button rather than the span, the buttons are announced correctly (refer to my attached demo). However, for styling purposes, assume the span elements must remain where they are, and the class cannot be moved to the button.

**Comment by graemecoleman** [comment]
Many apologies - after testing with JAWS, this appears to be a Firefox (related to how Firefox calculates the total size of the list), rather than an NVDA issue, so I'm going to close the ticket. For anyone interested, the (odd) solution is to apply role="presentation" to the span elements.
